docs/THIRDPARTY

```diff
@@ -113,6 +113,34 @@ Files: static/audio/notification_sounds/*
 Copyright: 2021 Stefan Mertens
 License: CC0-1.0
 
+Files: static/assets/icons/globe.svg
+Copyright: 2022 Ionic
+License: MIT License
+Comment: Increased the width of the globe outline.
+
+License: MIT License
+ The MIT License (MIT)
+ .
+ Copyright (c) 2015-present Ionic (http://ionic.io/)
```

Comment:
* Indicate the exact source of the original icon in the comment line (earth.svg from ionicons@5.5.2).
* According to the DEP-5 [specification](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification) for this format: “There are [many versions of the MIT license](https://en.wikipedia.org/wiki/MIT_License#Various_versions). Please use Expat instead, when it matches.”
* The license definitions are sorted by name, let’s keep them that way.
* Where does “2022 Ionic” come from?
* The copyright line is not part of the license.

Reply:
> Where does “2022 Ionic” come from?

From their license:

> Copyright (c) 2015-present Ionic (http://ionic.io/)


I switched the `MIT License` to `Expat`, since the text is the same, disregarding the copyright line. Is that what you wanted to say?

Reply:
You’ve quoted “2015-present Ionic (http://ionic.io/)”, but your patch lists “2022 Ionic”. Hence my question—where does “2022 Ionic” come from?

You can’t arbitrarily change the start date of a copyright. For example, the expiration date of copyright protection on a work made for hire is based on the creation or publication date.